src/utils/constants.ts

```
@@ -0,0 +1,59 @@
+  const hDisplay = h > 0 ? `${h} h ` : '';
+  const mDisplay = m > 0 ? `${m} m ` : '';
+  const sDisplay = s > 0 ? `${s} s` : '';
+  return dDisplay + hDisplay + mDisplay + sDisplay;
+}
+
+export function formatBytes(bytes: number): string {
+  if (bytes === 0) return '0B';
+  const k = 1024;
+  const sizes = ['B', 'KB', 'MB', 'GB', 'TB'];
+  const i = Math.floor(Math.log(bytes) / Math.log(k));
+  return `${parseFloat((bytes / Math.pow(k, i)).toFixed(2))}${sizes[i]}`;
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)